mysql_con.js

```diff
@@ -2,6 +2,9 @@ var mysql           = require("mysql");
 var express         = require("express");
 var app             = express();
 var bodyParser      = require("body-parser");
+var crypto          = require("crypto");
+var gets            = require("./app/mysql/get.js");
+var posts           = require("./app/mysql/post.js");
 
```

Comment:
You can definitely use GET to do this. GET can take any number of parameters, they just have to be in the parameter string:
/app/mysql/get.js?user_id=blah&password=sdflkjsdflkjsdflk&totems=suck